tests/cases/fourslash/getOutliningSpansForRegions.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference path="fourslash.ts"/>
+
+////// basic region
+////[|// #region
+////
+////// #endregion|]
+////
+////// region with label
+////[|// #region label1
+////
+////// #endregion|]
+////
+////// region with extra whitespace in all valid locations
+////[|             //              #region          label2    label3
+////
+////        //        #endregion|]
+////
+////// No space before directive
+////[|//#region label4
+////
+//////#endregion|]
+////
+////// Nested regions
+////[|// #region outer

```

Comment:
Is there any way to test the name that appears for the region? (And add some tests for interesting region names, e.g. includes spaces or tabs).